packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -93,7 +107,7 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     assert.equal(foundId, newFoundId);
   }
 
-  ['@htmlbars passing literal id is ignored and results in a default elementId'](assert) {
```

Comment:
test should just be titled that id is an alias for elementId, the title is massively confusing